.github/workflows/verify-pr-commit.yml

```diff
@@ -694,6 +694,9 @@ jobs:
       - name: Install NPM Modules
         run: npm ci
         working-directory: e2e
+      - name: Lint
+        run: npm run lint
+        working-directory: e2e
```

Comment:
Added to CI.

Sadly it is hard to run this separate from the e2e tests due to needing the api-augment package.